lib/iex/lib/iex/info.ex

```diff
@@ -1,6 +1,45 @@
 defprotocol IEx.Info do
   @fallback_to_any true
 
+  @moduledoc """
+  A protocol to print information in IEx about the given datastructure.
+
+  `IEx.Helpers.i/1` uses this protocol to display a term-specific list
+  of of information.
+
+  By default, an `Any` implementation will be used which returns
+  the `"Data type"`, `"Description"` and `"Reference modules"` sections.
+  """
+
+  @doc """
+  Returns information for the given term.
+
+  Information should be returned as a list of `info_name`-`info` pairs,
+
+  `info_name` should be short.
+  It is OK for `info` to contain newlines.
```

Comment:
```suggestion
  Information should be returned as a list of `info_name`-`info` tuples,
  where both values are strings. `info_name` should be short. `info` can
  be arbitrarily long and contain newlines.
```

Reply:
I'm for this change, but note that the spec:

```
  @spec info(term()) :: [{info_name :: String.Chars.t(), info :: String.t()}]
```

as well as [iex/helpers.ex:503](https://github.com/elixir-lang/elixir/blob/master/lib/iex/lib/iex/helpers.ex#L503) seem to hint that there are cases where `info_name` is not necessarily a string. 
Do we want to alter the spec and potentially remove the call to `to_string`?

Reply:
I updated the suggestion. Let me know what you think.